dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -38,7 +38,7 @@ service AppCallback {
   rpc OnBindingEvent(BindingEventRequest) returns (BindingEventResponse) {}
 }
```

Comment:
This should be targeted for 1.0 release branch instead of master

Reply:
I'll cut a 1.0 release branch as soon as this is merged. It was either merge to master and cut 1.0, or push 1.0 and merge to master. Same result in the end.